api/src/main/java/org/openmrs/Program.java

```diff
@@ -13,32 +13,59 @@
 import java.util.Locale;
 import java.util.Set;
 
+import jakarta.persistence.AttributeOverride;
+import jakarta.persistence.AttributeOverrides;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OrderBy;
+import jakarta.persistence.Table;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
 import org.hibernate.envers.Audited;
 import org.openmrs.annotation.AllowDirectAccess;
 
 /**
  * Program
  */
+@Entity
+@Table(name = "program")
 @Audited
+@AttributeOverrides({ @AttributeOverride(name = "name", column = @Column(name = "name", length = 100, nullable = true)) })
 public class Program extends BaseChangeableOpenmrsMetadata {
 	
 	public static final long serialVersionUID = 3214567L;
 	
 	// ******************
 	// Properties
 	// ******************
-	
+	@Id
+	@GeneratedValue(generator = "native")
+	@GenericGenerator(name = "native", strategy = "native", parameters = @Parameter(name = "sequence", value = "program_program_id_seq"))
```

Comment:
This generator is deprecated. Can you instead use this? https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Location.java#L63